pkg/cmd/repo/sync/sync.go

```diff
@@ -156,7 +156,7 @@ func syncLocalRepo(opts *SyncOptions) error {
 
 	if opts.IO.IsStdoutTTY() {
 		cs := opts.IO.ColorScheme()
-		fmt.Fprintf(opts.IO.Out, "%s Synced the \"%s\" branch from %s to local repository\n",
+		fmt.Fprintf(opts.IO.Out, "%s Synced the \"%s\" branch from \"%s\" to local repository\n",
```

Comment:
Another idea:

```suggestion
		fmt.Fprintf(opts.IO.Out, "%s Synced the \"%s\" local branch from \"%s\"\n",
```

Example:

```diff
 ✓ Synced the "muzimuzhi:trunk" branch from "cli:trunk"
-✓ Synced the "trunk" branch from cli/cli to local repository
+✓ Synced the "trunk" local branch from "cli/cli:trunk"
```

Reply:
Words are hard... 😅 Gonna dump some stream of consciousness, here:

What we're doing is taking the upstream branch and syncing our local branch to it. There are a few sentence constructions that convey that:

- Synced the "<upstream branch>" branch from upstream "<repo>" to the local "<branch>" branch
- Synced the local "<branch>" branch to the "<repo>:<upstream branch>" branch
- Synced the local "<branch>" branch with the "<repo>:<upstream branch>" branch
- Synced the "<repo>:<upstream branch>" branch with the local "<branch>" branch

I'm not sure your local branch needs to match the upstream name, so keeping the local branch name ambiguous is probably correct. That means all the options above that require <branch> don't work 😅 

Thus, the valid sentences are 

- Synced the "<upstream branch>" from "<repo>" to the local repository
- Synced the "<repo>:<upstream branch>" to the local repository
- Synced the local repository with the "<repo>:<upstream branch>"

ALL of this said, I think we leave it as it is.

And I feel a ton of empathy for those folks that aren't native english speakers... The English language is complicated 😅

Reply:
Sorry I can't fully understand what you meant, especially the one "That means all the options above that require don't work 😅". Perhaps because I'm one of non-native English speakers 😢

It seems you pointed out that names of local and remote branches just been in sync may differ, thus some sentence constructions are ambiguous.